src/styles/blocks/card.scss

```
@@ -0,0 +1,102 @@
+.card {
+  width: $card-width;
+  font-family: Roboto, sans-serif;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  color: $color-main-dark;
+  transition-property: transform;
+  transition-duration: 0.3s;
+
+  &:hover {
+    transition-duration: 0.3s;
+    transform: scale(1.2);
```

```suggestion
  &:hover {
    transform: scale(1.2);
```
There is no need to add transition for hover state